2025-10-21 08:15:59,353 : INFO : root : Loading config.
2025-10-21 08:15:59,358 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2025-10-21 08:15:59,359 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\fdca14d0c9d658ab8200d57120800af5.mpf_cache
2025-10-21 08:15:59,461 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-21 08:15:59,462 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\202a5840610e95051f9f473dfa660cc0.mpf_cache
2025-10-21 08:15:59,463 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\5dec96b1547c4adf5bdd3d1715e74bc0.mpf_cache
2025-10-21 08:15:59,463 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\2cd13a2d3f375e00ab01d6d09a85d8e7.mpf_cache
2025-10-21 08:15:59,464 : INFO : ConfigProcessor : Loading config from cache: C:\Users\antho\AppData\Local\Temp\9220b44b97b91972524500b81035d790.mpf_cache
2025-10-21 08:15:59,465 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\attract_lights.yaml
2025-10-21 08:15:59,465 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\attract_lights.yaml.
2025-10-21 08:15:59,483 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\attract_lights.yaml
2025-10-21 08:15:59,484 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\dab29d17bbe6500d014c71b0a57bd9b0.mpf_cache
2025-10-21 08:15:59,484 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\fade_out_graveyard_music.yaml
2025-10-21 08:15:59,486 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\fade_out_graveyard_music.yaml.
2025-10-21 08:15:59,496 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\fade_out_graveyard_music.yaml
2025-10-21 08:15:59,496 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\b17152efa55a24f0424602723f8d3921.mpf_cache
2025-10-21 08:15:59,496 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\fade_out_subsequentball.yaml
2025-10-21 08:15:59,496 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\fade_out_subsequentball.yaml.
2025-10-21 08:15:59,507 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\fade_out_subsequentball.yaml
2025-10-21 08:15:59,508 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\e87fc0e7d2f8a851b5fa3f604e0d0c6e.mpf_cache
2025-10-21 08:15:59,508 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\flash_all_lights.yaml
2025-10-21 08:15:59,508 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\flash_all_lights.yaml.
2025-10-21 08:15:59,517 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\flash_all_lights.yaml
2025-10-21 08:15:59,518 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\196ac76b6cab3a9880804c826cfbcb5c.mpf_cache
2025-10-21 08:15:59,518 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\graveyard_intro.yaml
2025-10-21 08:15:59,519 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\graveyard_intro.yaml.
2025-10-21 08:15:59,522 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\graveyard_intro.yaml
2025-10-21 08:15:59,522 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\834292dd7853e12a66732dc89828c828.mpf_cache
2025-10-21 08:15:59,523 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml
2025-10-21 08:15:59,523 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml.
2025-10-21 08:15:59,536 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\graveyard_lights.yaml
2025-10-21 08:15:59,536 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\ae2be9c5579852c305357e207df97e8b.mpf_cache
2025-10-21 08:15:59,537 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\graveyard_reach_house.yaml
2025-10-21 08:15:59,537 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\graveyard_reach_house.yaml.
2025-10-21 08:15:59,539 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\graveyard_reach_house.yaml
2025-10-21 08:15:59,540 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\46f8fa3e41dfeeb8232af2f03a13b6c4.mpf_cache
2025-10-21 08:15:59,540 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\house_lights.yaml
2025-10-21 08:15:59,540 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\house_lights.yaml.
2025-10-21 08:15:59,541 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\house_lights.yaml
2025-10-21 08:15:59,543 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\c67e9c175da6c060302945929b27b878.mpf_cache
2025-10-21 08:15:59,543 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\play_firstballintro.yaml
2025-10-21 08:15:59,543 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\play_firstballintro.yaml.
2025-10-21 08:15:59,544 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\play_firstballintro.yaml
2025-10-21 08:15:59,545 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\1c965aadbb61a7e709e8884fad17ac0b.mpf_cache
2025-10-21 08:15:59,545 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\antho\nightpin\shows\.\play_subsequentball.yaml
2025-10-21 08:15:59,545 : INFO : ConfigProcessor : Loading config from file C:\Users\antho\nightpin\shows\.\play_subsequentball.yaml.
2025-10-21 08:15:59,554 : INFO : ConfigProcessor : Loading config: C:\Users\antho\nightpin\shows\.\play_subsequentball.yaml
2025-10-21 08:15:59,556 : INFO : ConfigProcessor : Config file cache created: C:\Users\antho\AppData\Local\Temp\60ca9d662df3e70e96f6d7ec0f4491aa.mpf_cache
2025-10-21 08:15:59,556 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev11
2025-10-21 08:15:59,556 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': False, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs\\2025-10-21-08-15-59-mpf-LAPTOP-MPGP6R69.log', 'pause': False, 'production': False, 'text_ui': False, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-21-08-15-59-mpf-LAPTOP-MPGP6R69.log'}
2025-10-21 08:15:59,556 : INFO : Machine : MPF path: C:\Users\antho\mpf\Lib\site-packages\mpf
2025-10-21 08:15:59,556 : INFO : Machine : Machine path: C:\Users\antho\nightpin
2025-10-21 08:15:59,556 : INFO : Machine : Platform: win32
2025-10-21 08:15:59,556 : INFO : Machine : Python executable location: C:\Users\antho\mpf\Scripts\python.exe
2025-10-21 08:15:59,556 : INFO : Machine : Python version: 3.12.7 (64-bit)
2025-10-21 08:15:59,558 : INFO : Machine : Initializing MPF...
2025-10-21 08:15:59,864 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2025-10-21 08:15:59,895 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2025-10-21 08:15:59,895 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11', 'prev_value': None, 'change': True}
2025-10-21 08:15:59,895 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev11, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2025-10-21 08:15:59,896 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.12.7', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-11-10.0.26100-SP0', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '11', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '10.0.26100', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': 'C:\\Users\\antho\\nightpin\\logs\\2025-10-21-08-15-59-mpf-LAPTOP-MPGP6R69.log', 'prev_value': None, 'change': True}
2025-10-21 08:16:00,072 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2025-10-21 08:16:00,111 : INFO : EventManager : Event: ======'drop_target_bank_right_bank_up'====== Args={}
2025-10-21 08:16:00,112 : INFO : EventManager : Event: ======'drop_target_bank_left_bank_up'====== Args={}
2025-10-21 08:16:00,112 : INFO : EventManager : Event: ======'init_phase_2'====== Args={}
2025-10-21 08:16:00,115 : ERROR : asyncio : Exception in callback EventManager._queue_task_done(<Task finishe...pdate,queue')>)
handle: <Handle EventManager._queue_task_done(<Task finishe...pdate,queue')>)>
Traceback (most recent call last):
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 56, in validate_config_entry
    raise e
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 52, in validate_config_entry
    self._validate_config_item(device, device_settings))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 78, in _validate_config_item
    device_settings = self._parse_config(device_settings, device)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 144, in _parse_config
    return self.get_full_config(config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 216, in get_full_config
    return self.machine.config_validator.validate_config(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 164, in validate_config
    return self._validate_config(config_spec, source, base_spec=base_spec, add_missing_keys=add_missing_keys,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 204, in _validate_config
    processed_config[k] = self.validate_config_item(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 243, in validate_config_item
    return self.validate_item(item, validation, validation_failure_info)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 699, in validate_item
    return self.validator_list[validator](item, validation_failure_info=validation_failure_info, param=param)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 390, in _validate_type_enum
    raise self.validation_error(
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Failed to load config player graveyard_reach_house:shows with settings {'action': 'play', 'loops': 1} >> Config validation error: Entry show_player:show_player:action = "1" is not valid. Entry "1" is not valid for enum. Valid values are: play,stop,pause,resume,advance,step_back,update,queue Error Code: CFE-ConfigValidator-5 (https://missionpinball.org/logs)
2025-10-21 08:16:00,214 : INFO : Machine : Starting the main run loop with active modes: []
2025-10-21 08:16:00,214 : WARNING : Machine : Attract mode is not active, game will not be playable. Please check your attract mode configuration.
2025-10-21 08:16:00,215 : INFO : Machine : Shutting down...
2025-10-21 08:16:00,215 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2025-10-21 08:16:01,231 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\machine.py", line 810, in _run_loop
    raise self._exception['exception']
  File "C:\Users\antho\AppData\Local\Programs\Python\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 795, in _queue_task_done
    future.result()
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 56, in validate_config_entry
    raise e
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 52, in validate_config_entry
    self._validate_config_item(device, device_settings))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\config_players\device_config_player.py", line 78, in _validate_config_item
    device_settings = self._parse_config(device_settings, device)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 144, in _parse_config
    return self.get_full_config(config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_player.py", line 216, in get_full_config
    return self.machine.config_validator.validate_config(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 164, in validate_config
    return self._validate_config(config_spec, source, base_spec=base_spec, add_missing_keys=add_missing_keys,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 204, in _validate_config
    processed_config[k] = self.validate_config_item(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 243, in validate_config_item
    return self.validate_item(item, validation, validation_failure_info)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 699, in validate_item
    return self.validator_list[validator](item, validation_failure_info=validation_failure_info, param=param)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\antho\mpf\Lib\site-packages\mpf\core\config_validator.py", line 390, in _validate_type_enum
    raise self.validation_error(
mpf.exceptions.config_file_error.ConfigFileError: Config File Error in ConfigValidator: Failed to load config player graveyard_reach_house:shows with settings {'action': 'play', 'loops': 1} >> Config validation error: Entry show_player:show_player:action = "1" is not valid. Entry "1" is not valid for enum. Valid values are: play,stop,pause,resume,advance,step_back,update,queue Error Code: CFE-ConfigValidator-5 (https://missionpinball.org/logs)
2025-10-21 08:16:01,252 : INFO : root : MPF run loop ended.
